src/qt/statusbar.cpp

```diff
@@ -102,6 +102,11 @@ bool wxStatusBar::GetFieldRect(int i, wxRect& rect) const
     wxCHECK_MSG( (i >= 0) && ((size_t)i < m_panes.GetCount()), false,
                  "invalid statusbar field index" );
 
+    const_cast<wxStatusBar *>(this)->UpdateFields();
+
```

Comment:
`UpdateFields` is needed to initialize `m_qtPanes` (see the other comment)

Calling `UpdateFields` here seems ok because other ports initialize data in this method too. I don't know if there exists a better place to put it.